### ISSUE TITLE
Test apps don't work

### ISSUE THREAD

**Issue by piotrpospiech** [meteor/react-packages#371]
I tried to add a test to the `react-meteor-data` package, but I couldn't start tests. Test apps in the `/tests` directory are outdated and couldn't be started due to old versions of Meteor and packages. I get the `CERT_HAS_EXPIRED` error when I try to run them.

Is there another way to test these packages?

**Comment by radekmie** [comment]
I think the best would be to simply set it up again, with a newer Meteor version 🤷

**Comment by Grubba27** [comment]
I agree with @radekmie, is going to be less effort to create/rewrite the test suite using a newer version
